src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-details/osd-details.component.html

```diff
@@ -17,9 +17,9 @@
     </cd-table-performance-counter>
   </tab>
   <tab heading="Histogram">
-    <h3 *ngIf="osd.loaded && osd.histogram_failed">
-      Histogram not available -> <span class="text-warning">{{ osd.histogram_failed }}</span>
-    </h3>
+    <h4 class="alert alert-warning" *ngIf="osd.loaded && !osd.histogram_succeeded">
```

Comment:
you can use the new name of `histogram_failed`, no need to create a new variable for this.